.github/workflows/canary.yml

```
@@ -0,0 +1,48 @@
+name: canary
+on:
+  - pull_request
+  - push
+jobs:
+  canary:
+    name: testing ${{matrix.package}} with ${{matrix.node}} on ${{matrix.os}}
```

(see later for more info, but: I think testing only on `ubuntu-latest` and `node` is already useful, and I don’t see much added benefits by checking older node too).

```suggestion
    name: ${{matrix.package}}
```

Name is quite long and can’t be displayed properly (sidebar here: https://github.com/micromark/micromark/runs/4620836214?check_suite_focus=true)